tests/functional/package_test/__init__.py

```diff
@@ -37,6 +37,8 @@
 # Stopping event for CodeChecker server.
 __STOP_SERVER = multiprocessing.Event()
 
+__shared = {}
```

Comment:
The test infrastructure is completely rewritten, with different test layout/setup and configuration. For easier merge maybe we should split the testing part into a separate pull request.